docs/getting-started-on-snowplow-open-source/quick-start/index.md

```diff
@@ -413,7 +421,7 @@ CREATE SCHEMA IF NOT EXISTS ${schema_name}
 
 The security principal used by the loader needs a `Databricks SQL access` permission, which can be enabled in the _Admin Console_.
 
-Databricks does not have table access enabled by default. Enable it with an 
+Databricks does not have table access enabled by default. Enable it with an
```

Comment:
I know it's not the main point of your PR.  But.... you removed a space from a nearby sentence, so here goes.....

This section refers to a Databricks feature [which is described on their docs as "legacy"](https://docs.databricks.com/en/data-governance/table-acls/table-acl.html).  I wonder if we can drop this initialization script from our docs.  And say if users are on a legacy setup then they just need to figure this out for themselves.

Reply:
I think so yes.  But ideally we should test out this section end-to-end, with that part excluded.  Nobody currently still at Snowplow has ever run these steps.

Sorry to hijack your PR which is about something different!  We shouldn't let this be a blocker to merging the current PR.  But I also don't want to forget about it entirely.